### DISCUSSION TITLE
Dynamically set overview zoom levels

### DISCUSSION THREAD

**Discussion by eliabieri** [protomaps/PMTiles#425] [General]
## Overview
I'm developing an offline first application that shows POIs on a map. I want to create a `.pmtiles` extract that contains a certain area of each POI but also provide the user with an overview (on larger zoom levels) while having the smallest file size possible.

## Steps I've taken
1. Create a GeoJSON MultiPolygon where each polygon is a circle with a certain radius around every POI
2. Extract a `.pmtile` file using the `pmtiles` utility.

## Issues I'm facing
- The overview zoom levels defaults lead to empty spots in the map.
  - The POIs are all located in southern Germany
  - I want an overview map up to a certain zoom level to be available for this whole region while having the full zoom dataset for areas around POIs

I appreciate every idea on how I could get this problem solved.

**Comment by bdon** [comment]
(Moved to discussion)

How are you creating your tileset from GeoJSON? You may want to use https://github.com/felt/tippecanoe with advanced options and remove all unnecessary features to pack more POIs in overview levels.
